### ISSUE TITLE
Gradle: Cannot start MARY server

### ISSUE THREAD

**Issue by cniesen** [marytts/marytts-txt2wav#8]
I'm having a hard time running the gradle sample.  I get the following error:
```
C:\marytts-txt2wav>java -jar build/libs/txt2wav-1.0-SNAPSHOT.jar -i test.txt -o text.wav
Could not initialize MaryTTS interface: Cannot start MARY server
Exception in thread "main" marytts.exceptions.MaryConfigurationException: Cannot start MARY server
        at marytts.LocalMaryInterface.<init>(LocalMaryInterface.java:66)
        at de.dfki.mary.Txt2Wav.main(Txt2Wav.java:83)
Caused by: java.lang.StringIndexOutOfBoundsException: begin 0, end 3, length 2
        at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3107)
        at java.base/java.lang.String.substring(String.java:1873)
        at marytts.server.EnvironmentChecks.check(EnvironmentChecks.java:46)
        at marytts.server.Mary.startup(Mary.java:286)
        at marytts.server.Mary.startup(Mary.java:204)
        at marytts.util.MaryRuntimeUtils.ensureMaryStarted(MaryRuntimeUtils.java:72)
        at marytts.LocalMaryInterface.<init>(LocalMaryInterface.java:64)
        ... 1 more
```
I did see the https://github.com/marytts/marytts/issues/690 issue but didn't quite get what the problem/fix was.  In my build the txt2wav-1.0-SNAPSHOT.jar/META-INF/services/marytts.config.MaryConfig file contains:
```
marytts.voice.CmuSltHsmm.Config
marytts.language.en.EnglishConfig
marytts.config.MainConfig
```
Any clue what might be wrong?

BTW, I updated the sample project.  Specifically I needed to update the gradle wrapper version since the old one didn't build the project for me.
Pull request: https://github.com/marytts/marytts-txt2wav/pull/7

**Comment by seblemaguer** [comment]
Hello,

it is a problem with a java version. It is solved in the master branch of mary but we have to publish the artefact.

**Comment by GABowers** [comment]
> 
> 
> Hello,
> 
> it is a problem with a java version. It is solved in the master branch of mary but we have to publish the artefact.

Was said artefact ever published? I'm getting the same issue with the maven version of this project.

**Comment by thomas-genet** [comment]
Hi. I had the same problem when switching from Java 8 to Java 11. Since I badly need the version of marytts that I have (and love!) I needed to find a workaround. I found one which is really dirty but worked for me. I call `new LocalMaryInterface()` twice. The first call is protected by a try catch that only calls the constructor a second time... which succeeds surprisingly. I hope this helps people waiting for a fixed version.
